src/planscape/planning/serializers.py

```diff
@@ -425,7 +439,7 @@ class ProjectAreaSerializer(serializers.ModelSerializer):
     class Meta:
         model = ProjectArea
         fields = (
-            "uuid",
+            "id",
```

Comment:
we had two instances of ProjectAreaSerializer somehow, maybe from a bad merge. One was being ignored.